src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/DefaultRazorLanguageServerCustomMessageTarget.cs

```diff
@@ -475,5 +476,54 @@ private static bool SupportsCSharpCodeActions(JToken token)
 
             return Task.FromResult(result.ToArray());
         }
+
+        public override async Task<VSInternalWrapWithTagResponse> RazorWrapWithTagAsync(VSInternalWrapWithTagParams wrapWithParams, CancellationToken cancellationToken)
+        {
+            // Same as in LanguageServerConstants, and in Web Tools
+            const string HtmlWrapWithTagEndpoint = "textDocument/_vsweb_wrapWithTag";
+
+            var response = new VSInternalWrapWithTagResponse(wrapWithParams.Range, Array.Empty<TextEdit>());
+
+            var hostDocumentUri = wrapWithParams.TextDocument.Uri;
+            if (!_documentManager.TryGetDocument(hostDocumentUri, out var documentSnapshot))
+            {
+                return response;
```

Comment:
Receiving requests for documents we're not aware of seems worthy of logging to me.